src/code/z_msgevent.c

```diff
@@ -1,20 +1,19 @@
 #include "global.h"
 
 void MsgEvent_SendNullTask(void) {
-    s32 pad[4];
     OSScTask task;
     OSMesgQueue queue;
     OSMesg msg;
     u32 pad2[1];
 
     task.next = NULL;
     task.flags = OS_SC_RCP_MASK;
-    task.msgQ = &queue;
+    task.msgQueue = &queue;
     task.msg = NULL;
     task.framebuffer = NULL;
     task.list.t.type = M_NULTASK;
-    osCreateMesgQueue(task.msgQ, &msg, 1);
-    osSendMesg(&gSchedContext.cmdQ, &task, OS_MESG_BLOCK);
+    osCreateMesgQueue(task.msgQueue, &msg, 1);
```

Comment:
Both are equally valid, I personally prefer this when trying to receive exactly one message.

Reply:
Yeah there's no strict convention so I've opted to stick with not using arrays for single messages since that's what we already do. One notable exception is `siacs.c` and `piacs.c`, this is just to match the original libultra types.